README.md

```diff
@@ -145,21 +150,31 @@ yarn workspace discovery-search-app run start
      - `DISCOVERY_URL` is the API URL to your Discovery instance (ex. `https://api.us-south.discovery.cloud.ibm.com/instances/2386cfd4-a584-41d0-868d-671d8be819bc`)
      - `DISCOVERY_APIKEY` the api key associated with `DISCOVERY_URL`
 
+   <img src="./docs/images/example-url-apikey.png" alt="URL and API key" width="800px">
+
+
 6. Build the React components:
 
    ```
    yarn workspace @ibm-watson/discovery-react-components run build
    ```
 
-7. Start the example app:
+7. Start the example server:
+
+   ```
+   yarn workspace discovery-search-app server
```

Comment:
Is this correct?
```suggestion
   yarn workspace discovery-search-app run server
```

Reply:
`yarn server` or `yarn run server` seem to run the same, but I can update to `yarn run server`.